m4/ax-python-module.m4

```diff
@@ -4,13 +4,14 @@
 #
 # SYNOPSIS
 #
-#   AX_PYTHON_MODULE(modname[, fatal])
+#   AX_PYTHON_MODULE(module_name, min_version)
```

Comment:
Upon further reading I would suggest checking the version in a separate macro. It can be a very simple macro if you use this macro to determine that the module is present already. Also consider doing the comparison from within Python in order to gain access to a dedicated function for comparing versions.

Something like this:
```m4
AC_DEFUN([AX_PYTHON_MODULE_VERSION], [
    AX_PYTHON_MODULE([$1])
    AC_MSG_CHECKING([for version $2 of $1])
    echo "import sys, $1; from distutils.version import StrictVersion; sys.exit(StrictVersion($1.__version__) >= StrictVersion('$2'))" > conftest.py
    $PYTHON conftest.py >/dev/null 2>&1
    AS_IF([test $? -eq 0], [], [
        AC_MSG_RESULT([no])
        AC_MSG_ERROR([You need at least version $2 of the $1 Python module.])
    ])
    AC_MSG_RESULT([yes])
])
```